packages/paste-website/src/component-examples/DeletePatternExamples.ts

```diff
@@ -0,0 +1,119 @@
+export const LowSeverityExample = `
+  const LowSeverityDelete = () => {
+    return (
+      <Button variant="destructive">Remove</Button>
+    )
+  }
+  
+  render(
+    <LowSeverityDelete />
+  )
+`.trim();
+
+export const MediumSeverityExample = `
+  const MediumSeverityDelete = () => {
+    const [isOpen, setIsOpen] = React.useState(false);
+    const handleOpen = () => setIsOpen(true);
+    const handleClose = () => setIsOpen(false);
+    return (
+      <>
+        <Button variant="destructive" onClick={handleOpen}>Delete</Button>
+        <AlertDialog
+          heading="Delete document from this regulatory bundle"
+          isOpen={isOpen}
+          onConfirm={handleClose}
+          onConfirmLabel="Delete"
+          onDismiss={handleClose}
+          onDismissLabel="Cancel"
+          destructive
+        >
+          You're about to delete "Plan A Productions, LLC" from this regulatory bundle. This action does not impact your other 188 regulatory bundles.
+        </AlertDialog>
+      </>
+    )
+  }
+  render(
+    <MediumSeverityDelete />
+  )
+`.trim();
+
+export const HighSeverityExample = `
+  const HighSeverityDelete = () => {
+  const [isDisabled, setIsDisabled] = React.useState(true);
+  const [isOpen, setIsOpen] = React.useState(false);
+  const handleOpen = () => setIsOpen(true);
+  const handleClose = () => setIsOpen(false);
+  const handleChange = (e) => {
+    if (e.target.value === 'Toyota TCB Automobile (Gevelsberg)') setIsDisabled(false);
+    else setIsDisabled(true);
+  };
```

Comment:
todo: ideally, the interaction here is:

- The button becomes enabled when any text is entered in the field
- If the name is typed wrong, submitting it errors out.

Mainly because if a user typos and doesn't notice, it's confusing to not know why the button won't enable.